airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -27,6 +27,8 @@
 import os
 import tempfile
 
+from oauth2client.client import GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
isn't oauth2client library deprecated?

Reply:
Can we use https://google-auth.readthedocs.io/en/latest/ instead?

Reply:
I updated a PR. 
In addition, I added a new point to the list of changes:
> some tests were dependent on the database. I added a mock. In the future I will try to fix the constructor. It should not connect to the database